tests/python/pants_test/backend/python/tasks/test_select_interpreter.py

```diff
@@ -95,7 +101,7 @@ def se(me, *args, **kwargs):
     return interpreter.version_string
 
   def test_interpreter_selection(self):
-    self.assertEquals('IronPython-2.77.777', self._select_interpreter([]))
```

Comment:
It might make sense to add a test that asserts that no interpreter is created when there are no targets. I don't feel super strongly about it though.

Maybe something like


```
     empty_target_roots=[]
     context = self.context(target_roots=empty_target_roots)
     task = self.create_task(context)

     # Mock out the interpreter cache setup, so we don't actually look for real interpreters
     # on the filesystem.
     with mock.patch.object(PythonInterpreterCache, 'setup', autospec=True) as mock_resolve:
       def se(me, *args, **kwargs):
         me._interpreters = self.fake_interpreters
         return self.fake_interpreters
       mock_resolve.side_effect = se
       task.execute()
 
    self.self.assertIsNone(context.products.get_data(PythonInterpreter))
```

Reply:
Done